zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -105,6 +111,9 @@
   // ZkEventThread. Otherwise the retry request might block the normal event processing.
   protected final ZkAsyncRetryThread _asyncCallRetryThread;
 
+  private final boolean _syncOnNewSession;
+  private static final String _syncPath = "/";
```

Comment:
I think I have commented before the name convention of static final field is SYNC_PATH.

Reply:
fixed.